cli/azd/pkg/input/console.go

```diff
@@ -216,13 +280,20 @@ func (c *AskerConsole) ShowSpinner(ctx context.Context, title string, format Spi
 		return
 	}
 
+	if c.previewer != nil {
+		// spinner is not compatible with previewer.
+		c.previewer.Header(c.currentIndent + title)
+		return
+	}
```

Comment:
Why does the spinner now have to enact side effects on the previewer?

Reply:
b/c the spinner implemented within the AskerConsole needs to own the last line of the terminal, so it is not compatible to have this spinner running at the same time as the progress log.
For any component, running the next steps are fine:
console.ShowSpinner()
console.ShowProgressLog()
console.ShowSpinner()
console.StopProgressLog()